root/layout/components/ExternalLinks.js

```diff
@@ -94,7 +95,7 @@ const ExternalLinks = ({
     const relationship = relationships[i];
     const target = relationship.target;
 
-    if (relationship.ended || target.entityType !== 'url') {
+    if (isDisabledLink(relationship, target) || target.entityType !== 'url') {
```

Comment:
nitpick, let's reverse the order of these so the function doesn't need to be called on non-url relationships
```suggestion
    if (target.entityType !== 'url' || isDisabledLink(relationship, target)) {
```